.github/workflows/main.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ["3.9", "3.10", "3.12"]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
At this point, you might as well add `"3.13"`. Pre-release versions are already available in GitHub runners.